docs/examples/resnet50_trt/setup_resnet50_trt_example.sh

```diff
@@ -25,9 +25,9 @@ mkdir -p model_repository/dali/1
 mkdir -p model_repository/ensemble_dali_resnet50/1
 mkdir -p model_repository/resnet50_trt/1
 
-docker run -it --gpus=all -v $(pwd):/workspace nvcr.io/nvidia/pytorch:21.03-py3 /bin/bash -cx \
+docker run -it --gpus=all -v $(pwd):/workspace nvcr.io/nvidia/pytorch:25.03-py3 /bin/bash -cx \
```

Comment:
Is this version pin needed, or maybe using `latest` would be a better option? Maybe we should document this in the deps update procedure, so that somebody will bump it from time to time?

Reply:
This is a script that is not directly run by tests, so it doesn't matter that much. I don't think there's latest tag but I think it's ok if this serves as instruction that is not necessary fully up to date.